libs/platform/src/lib/components/form/text-area/text-area.component.html

```diff
@@ -0,0 +1,31 @@
+<textarea
+    #textareaElement
+    fd-form-control
+    [compact]="contentDensity == 'compact'"
+    [ngClass]="[growing ? 'fdp-textarea-custom-resize' : 'fdp-textarea-normal-size']"
+    [attr.id]="id"
+    [attr.placeholder]="placeholder"
+    [disabled]="disabled"
+    [attr.aria-disabled]="disabled"
+    [readonly]="readonly"
+    [attr.aria-readonly]="readonly"
+    [rows]="growing ? 2 : height ? 2 : growingMaxLines"
+    [cols]="cols"
+    [wrap]="wrapType"
+    [maxlength]="!showExceededText ? maxLength : null"
+    [(ngModel)]="value"
+    [state]="getUpdatedState()"
+    (paste)="handlePasteInteraction()"
+    (blur)="_onFocusChanged(false)"
+    (focus)="_onFocusChanged(true)"
+></textarea>
+
+<!-- ICU recommends full text in format -->
+<div class="fd-textarea-counter" *ngIf="showExceededText" role="alert" aria-atomic="true">
```

Comment:
Let's first figure out my comment above about i18n

Reply:
Please check my reply above on i18n https://github.com/SAP/fundamental-ngx/pull/2691#discussion_r445471699. 
If it doesn't make much sense, I can remove it from this PR. I would also probably have to remove the `Support for i18n of message counter` as a platform enhancement description in this case.